build_tools/python/e2e_test_framework/models/jax_models.py

```diff
@@ -86,3 +86,16 @@
     ],
     batch_sizes=[1, 16, 24, 32, 48, 64, 512],
 )
+
+# Derived from https://huggingface.co/docs/transformers/model_doc/gemma#transformers.FlaxGemmaForCausalLM
+GEMMA_TAGS = ["gemma", "decoder-only", "pipeline", "generative"]
+
+GEMMA_2B_IT_GREEDY_FP32_JAX_1024XI32_256I32 = common_definitions.Model(
+    name="Gemma2bit_fp32",
+    id=unique_ids.MODEL_GEMMA_2B_IT_GREEDY_FP32_JAX_1024XI32_256I32,
+    tags=GEMMA_TAGS + ["fp32"],
+    source_type=common_definitions.ModelSourceType.EXPORTED_STABLEHLO_MLIR,
+    source_url="https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.25_1709787220/GEMMA2BIT_GREEDY_FP32_JAX_1X1024XI32_256XI32/stablehlo.mlirbc",
+    entry_function="main",
+    input_types=["1x1024xi32"],
```

Comment:
Looks like this failed with
```cmd: [PosixPath('/work/linux-x86_64-benchmark-tools-dir/build/tools/iree-benchmark-module'), '--module=/tmp/iree-benchmarks/97adcc6379d65b94ca854da90290e4a38f62ca40/iree_module_Gemma2bit_fp32_stablehlo___x86_64-cascadelake-linux_gnu-llvm_cpu__default-flags_dt-uk_/module.vmfb', '--function=main', '--device_allocator=caching', '--task_topology_max_group_count=30', '--device=local-task', '--input=1x1024xi32=0', '--time_unit=ns', '--benchmark_format=json', '--benchmark_out_format=json', '--print_statistics=true', '--benchmark_repetitions=10']```

```work/runtime/src/iree/tooling/function_io.c:387: UNIMPLEMENTED; unimplemented cconv type `v`; parsing input `1x1024xi32=0```
https://github.com/iree-org/iree/actions/runs/9691839363/job/26744473157?pr=17749#step:9:46

Reply:
Fixed that error but now running into a new one: https://github.com/iree-org/iree/issues/17774